projects/fsastorefrontlib/src/core/date-config/default-date-config.ts

```diff
@@ -0,0 +1,12 @@
+import { FormDateConfig } from '@fsa/dynamicforms';
+import { DateConfig } from './date-config';
+
+export function fsDefaultDateFormatConfigFactory():
+  | DateConfig
+  | FormDateConfig {
```

Comment:
Do we need both of these configs since they are the same?

Reply:
I wanted to be sure that it works for both cases. Somehow it works with this DateConfig too, although I use FormDateConfig as dependency injection in components.